libpod/sql_state.go

```diff
@@ -135,6 +137,8 @@ func (s *SQLState) Refresh() (err error) {
 		return errors.Wrapf(err, "error refreshing database state")
 	}
 
+	committed = true
```

Comment:
Should this be after the next if statement?  If tx.Commit has an error, will it be commited?  Same comment through out.

Reply:
OK, good to know, thanks!